code/modules/modular_computers/computers/item/computer.dm

```diff
@@ -984,6 +988,67 @@
 		return stored_files
 	return stored_files + inserted_disk.stored_files
 
+/// Returns how relevant the current security level is:
+#define ALERT_RELEVANCY_SAFE 0 /// * 0: User is not in immediate danger and not needed for some station-critical task.
+#define ALERT_RELEVANCY_WARN 1 /// * 1: Danger is around, but the user is not directly needed to handle it.
+#define ALERT_RELEVANCY_PERTINENT 2/// * 2: Danger is around and the user is responsible for handling it.
+/obj/item/modular_computer/proc/get_security_level_relevancy(var/obj/item/card/id/idToCheck)
+	var/list/id_accesses = list()
+	id_accesses = saved_id_accesses
+
+	switch(SSsecurity_level.get_current_level_as_number())
+		if(SEC_LEVEL_DELTA)
+			return ALERT_RELEVANCY_PERTINENT
+		if(SEC_LEVEL_RED) // all-hands-on-deck situations, everyone is responsible for combatting a threat
+			return ALERT_RELEVANCY_PERTINENT
+		if(SEC_LEVEL_BLUE) // suspected threat. security needs to be alert and possibly preparing for it, no further concerns
+			if(ACCESS_SECURITY in saved_id_accesses)
+				return ALERT_RELEVANCY_PERTINENT
+			else
+				return ALERT_RELEVANCY_SAFE
+		if(SEC_LEVEL_GREEN) // no threats, no concerns
+			return ALERT_RELEVANCY_SAFE
+
+	return 0
```

Comment:
This should be a proc on the security level datum itself, makes it easier to add new security levels to this